AUTOMATAFORMAT.md

```diff
@@ -45,12 +45,12 @@ We use the general form of AFA, by Pavol V., and simple NFA.
 * `true` means true and `false` means false.
 
 ## Alphabet type
-Next, we categorise automata by the **alphabet type**, i.e., the representation of symbols on transitions. We will consider propositional formulae (over bit vectors), explicit symbols, intervals of numbers or unicode or ascii, or user-defined alphabet. The type of alphabet is encoded in a name of section, e.g., `@NFA-explicit` denotes that nondeterministic finite automaton over explicit alphabet is given.
+Next, we categorise automata by the **alphabet type**, i.e., the representation of symbols on transitions. We will consider propositional formulae (over bit _vectors), explicit symbols, intervals of numbers or unicode or ascii, or user-defined alphabet. The type of alphabet is encoded in a name of section, e.g., `@NFA-explicit` denotes that nondeterministic finite automaton over explicit alphabet is given.
```

Comment:
Again, it looks like an unwanted replace all.

Reply:
This is shit, plus the shit with git, argh, I will better make a new branch and coppy my changes there and make another pull request. I will revert the name changes there, so that we can merge it now quickly, and we will decide on how to name things somehow soon.